src/whisper_at_train/run_combined_training.sh

```
@@ -7,49 +11,83 @@
-#label_csv=/mnt/ssd_disk/github/whisper-at/audioset_label.csv
 n_class=533
-# n_class=527
 label_smooth=0.1
 
-pretrained_model=/mnt/ssd_disk/github/whisper-at/pretrained_models/large-v1_ori.pth
-
 # Get current timestamp in YYMMDDHHMM format
 timestamp=$(date +%y%m%d%H%M)
 
 exp_dir=./exp/combined-ft-${dataset}-${model}-${model_size}-${lr}-${lrscheduler_start}-${lrscheduler_decay}-ep${epoch}-bs${batch_size}-lda${lr_adapt}-ls${label_smooth}-mix${mixup}-${freqm}-${timem}-${timestamp}
 mkdir -p $exp_dir
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)